.codeclimate.yml

```diff
@@ -0,0 +1,6 @@
+languages:
+  JavaScript: true
+
+exclude_paths:
+- "database_reports/*"
+- "migrations/*"
```

Comment:
I checked the analysed files here: https://codeclimate.com/github/HabitRPG/habitrpg/code?page=3

The test directory is not included. I don't know if code climate excludes that folder by default or if some other black magic goes on in the background ;)

I think it should be fine if we leave it as is. I think it could be useful to know if there are any issues in the tests at some point. Then it could still be decided to exclude that folder in the yml. What do you think, @crookedneighbor?